docs/source/AdministratorGuide/Resources/identityprovider.rst

```diff
@@ -0,0 +1,59 @@
+.. _resourcesIdProvider:
+
+=================
+Identity Provider
+=================
+
+An identity provider (IdP) is a system entity that creates, maintains, and manages identity information for principals and also provides authentication services to relying applications within a federation or distributed network.
+IdP offer user authentication as a service. DIRAC has its own user base, but it is not an IdP as a service, therefore, it uses third-party IdP for authorization.
+
+These resources are used by the following DIRAC instances:
+
+- Framework/TokenManager service
+- Framework/Auth endpoint
```

Comment:
"endpoint" because it is only http?

Reply:
There are two type of instances now:

- Services 
  * http/diset
  * use post
  * request url like as https://server:8443/< system >/< service > + {'method': **methodName**, 'args': {**arg**:**option**}}
- APIs (endpoints)
  * only http
  * use post, get, etc. http methods
  * request url like as https://server/any/path/**methodName**/canBePositionalOption?**arg**=**option**

Reply:
Maybe https://dirac.readthedocs.io/en/latest/AdministratorGuide/Introduction/diraccomponents.html should be expanded then.

Reply:
Ill describe it here https://github.com/DIRACGrid/DIRAC/pull/5400